src/tests/test_dynamic_api/test_views_wfs.py

```diff
@@ -112,3 +113,47 @@ def test_wfs_feature_name(self, api_client, afval_dataset, afval_adresloopafstan
             "geometry": None,
             "serienummer": "foobar-456",
         }
+
+    def test_wfs_default_dataset_exposed(
+        self,
+        api_client,
+        router,
+        bommen_dataset,
+    ):
+        """Prove that if feature name contains non-letters like underscore,
+        it can be useds find the correct table name and data
+        """
+        router.reload()
+        # manually creating tables, as we do not want to use `filled_router` here.
```

Comment:
why? :-)  (perhaps useful to add in the comment)

Reply:
I'll  copy paste comment from another test. Good one.